src/gql/generated/types.ts

```diff
@@ -104,6 +111,49 @@ export type Host = {
   uptime?: Maybe<Scalars["Time"]>;
   elapsed?: Maybe<Scalars["Time"]>;
   startedBy: Scalars["String"];
+  provider: Scalars["String"];
+  lastCommunicationTime?: Maybe<Scalars["Time"]>;
+  noExpiration: Scalars["Boolean"];
+  instanceType?: Maybe<Scalars["String"]>;
+  homeVolumeID?: Maybe<Scalars["String"]>;
+  user?: Maybe<Scalars["String"]>;
+  distro?: Maybe<DistroInfo>;
+  availabilityZone?: Maybe<Scalars["String"]>;
+  instanceTags?: Maybe<Array<Maybe<InstanceTag>>>;
+  expiration?: Maybe<Scalars["Time"]>;
+};
+
+export type HostEventLogData = {
+  agentRevision: Scalars["String"];
+  agentBuild: Scalars["String"];
+  jasperRevision: Scalars["String"];
+  oldStatus: Scalars["String"];
+  newStatus: Scalars["String"];
+  logs: Scalars["String"];
+  hostname: Scalars["String"];
+  provisioningMethod: Scalars["String"];
+  taskId: Scalars["String"];
+  taskPid: Scalars["String"];
+  taskStatus: Scalars["String"];
+  execution: Scalars["String"];
+  monitorOp: Scalars["String"];
+  user: Scalars["String"];
+  successful: Scalars["Boolean"];
+  duration: Scalars["Duration"];
+};
+
+export type HostEventLogEntry = {
```

Comment:
Have you tried merging master recently? I added a HostEventLogEntry like 5 days ago.

Reply:
I have, This file is autogenerated from the schema.graphql file and the queries. It likely just regenerated parts of this file and the git diff caught this as a new change. I double checked the file these fields weren't redeclared twice